### ISSUE TITLE
Accessibility - Form control element <input> has no associated label

### ISSUE THREAD

**Issue by kyjak** [carbon-design-system/carbon-for-ibm-dotcom#6116]
the search input box has no label
reporting url https://ibmdocs-dev.mybluemix.net/docs/en/error
you can use the ibm equal access checker

`<input value="" name="q" data-autoid="dds--header__search--input" placeholder="Search IBM Documentation" class="bx--header__search--input" aria-controls="react-autowhatever-1" aria-autocomplete="list" autocomplete="off" type="text">
`

**Comment by RobertaJHahn** [comment]
Kenny will review and document more details for the next review.

**Comment by kennylam** [comment]
Confirmed fix is needed for React version.
